src/EA.Weee.Web/Areas/AatfReturn/Views/SentOnSiteSummaryList/index.cshtml

```diff
@@ -17,76 +17,94 @@
                         </div>
-                        <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display">
-                            Actions
+                        <div class="govuk-grid-column-one-half">
+                            <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display" style="word-break:keep-all">
+                                B2C (tonnes)
+                            </div>
+                            <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display" style="word-break:keep-all">
+                                B2B (tonnes)
+                            </div>
+                            <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display">
+                                Actions
+                            </div>
                         </div>
                     </div>
                 </div>
-            </div>
-            foreach (var address in Model.Sites)
-            {
-                <div class="govuk-grid-row govuk-table__cell">
-                    <div class="govuk-grid-column-one-quarter weee-tonnage-column-padding-left" style="word-break:break-all">
-                        @address.SiteAddress.Name<br />
-                        @address.SiteAddress.Address1<br />
-                        @address.SiteAddress.Address2<br />
-                        @address.SiteAddress.TownOrCity<br />
-                        @address.SiteAddress.CountyOrRegion<br />
-                        @address.SiteAddress.Postcode<br />
-                        @address.SiteAddress.CountryName<br />
-                    </div>
-                    <div class="govuk-grid-column-one-quarter weee-tonnage-column-padding-left" style="word-break:break-all">
-                        @address.OperatorAddress.Name<br />
-                        @address.OperatorAddress.Address1<br />
-                        @address.OperatorAddress.Address2<br />
-                        @address.OperatorAddress.TownOrCity<br />
-                        @address.OperatorAddress.CountyOrRegion<br />
-                        @address.OperatorAddress.Postcode<br />
-                        @address.OperatorAddress.CountryName<br />
-                    </div>
-                    <div class="govuk-grid-column-one-half">
-                        <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display">
-                            @address.Tonnages.B2C
+                foreach (var address in Model.Sites)
+                {
+                    <div class="govuk-grid-row govuk-table__cell">
+                        <div class="govuk-grid-column-one-quarter weee-tonnage-column-padding-left" style="word-break:break-all">
+                            @address.SiteAddress.Name<br />
+                            @address.SiteAddress.Address1<br />
+                            @if (!string.IsNullOrEmpty(address.SiteAddress.Address2))
```

Comment:
Would it be possible to generate the site/operator address string with a function in the controller? I've pretty much done this in the ReusedRemoveSiteController, just to make the view a bit cleaner and we have something to test

Reply:
Good shout. Will do that.

Reply:
Not been able to do it in the controller, but added it to the model.